### PR TITLE
update regression tests guide

### PR DESCRIPTION
**Pull request by luarss**
image

@vvbandeira

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,29 +257,45 @@ from [here](docs/user/Build.md).
...
+
 # run <tool> tests
-test/regression <tool>
-# run <tool> tool tests
-src/<tool>/test/regression
+./test/regression <tool>
+
+# run <tool>-specific unit tests
+cd src/<tool>
+./test/regression
```
Comment (vvbandeira) [inline README.md:274]:
```suggestion
# run all <tool>-specific unit tests
cd src/<tool>
./test/regression

# run only <TEST_NAME> for <tool>
cd src/<tool>
./test/regression <TEST_NAME>
```

File: README.md [hunk 2]
```diff
@@ -257,29 +257,45 @@ from [here](docs/user/Build.md).
...
 ``` text
 % report_flow_metrics gcd_nangate45
                        insts    area util slack_min slack_max  tns_max clk_skew max_slew max_cap max_fanout DPL ANT drv
 gcd_nangate45            368     564  8.8     0.112    -0.015     -0.1    0.004        0       0          0   0   0   0
 ```
 
+To update a failing regression, follow the instructions below:
+
+```tcl
+# update log files (i.e. *.ok)
```
Comment (vvbandeira) [inline README.md:289]:
```suggestion
# update log files (i.e. *ok)
```

File: README.md [hunk 3]
```diff
@@ -257,29 +257,45 @@ from [here](docs/user/Build.md).
 
 ## Regression Tests
 
-There are a set of regression tests in `test/`.
+There are a set of executable regression test scripts in `test/`.
```
Comment (vvbandeira) [inline README.md:260]:
```suggestion
There are a set of executable regression test scripts in `./test/`.
```

File: README.md [hunk 4]
```diff
@@ -257,29 +257,45 @@ from [here](docs/user/Build.md).
 
 ## Regression Tests
 
-There are a set of regression tests in `test/`.
+There are a set of executable regression test scripts in `test/`. 
 
 ``` shell
-# run all tool unit tests
-test/regression
+# run all tests
```
Comment (vvbandeira) [inline README.md:263]:
```suggestion
# run tests for all tools
```

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Review by vvbandeira — CHANGES_REQUESTED** [review]
See above, I re-opened one as it was not fully addressed.

**Comment by luarss** [comment]
Ok, should reflect the changes now.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**